packages/@react-spectrum/s2/intl/en-US.json

```diff
@@ -12,6 +12,7 @@
   "label.(optional)": "(optional)",
   "menu.moreActions": "More actions",
   "picker.placeholder": "Select…",
+  "segmentedcontrol.aria-label": "Segmented control",
```

Comment:
Not blocking: I think we should require an `aria-label` for this component, or choose a default that is more understandable.

Reply:
Agreed, I have no idea what a 'Segmented control' is, despite having been in many meetings about them. So we'll want to ask Spectrum for a default or require it.

Also, I think we should name the string a little differently `segmentedcontrol.defaultlabel` or `segmentedcontrol.defaultname` or something.

Finally, I don't think I have logic to handle strings only existing in s2 in my migrate intl script. If you have some cycles, maybe see about adding that. Otherwise, I can handle it soon.

Reply:
yeah I don't think we need a default label, we should just require one like we did before in ActionGroup

Reply:
sounds good, will update it to be required